src/kernels/raw/finder/interpreterKernelSpecFinderHelper.node.ts

```diff
@@ -34,7 +34,10 @@ import { createPromiseFromCancellation } from '../../../platform/common/cancella
 import { sendTelemetryEvent } from '../../../telemetry';
 import { getTelemetrySafeHashedString } from '../../../platform/telemetry/helpers';
 
-export const LocalPythonKernelsCacheKey = 'LOCAL_KERNEL_PYTHON_AND_RELATED_SPECS_CACHE_KEY_V_2023_3';
+export function localPythonKernelsCacheKey() {
```

Comment:
This doesn't really need to be a function does it? It could be just an exported constant?